zulipterminal/ui.py

```diff
@@ -77,6 +77,26 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Tuple[Any]],
+                             suggestion: Optional[str]) -> None:
+        # wrap by space and use only name from tuple
+        suggestions = list(map(lambda x: ' ' + x[0] + ' ', suggestions))
+
+        if suggestion is not None:
+            # FIXME deal with suggestions having same names
```

Comment:
This is likely where we'll start to include ids, and those can be included in at least mentions. I'm not sure if there is ambiguity in other cases?

Reply:
There isn't any in emojis, and I doubt there would be much in streams either.